pkg/apis/batch/validation/validation_test.go

```diff
@@ -120,8 +122,9 @@ func TestValidateJob(t *testing.T) {
 			job: batch.Job{
 				ObjectMeta: validJobObjectMeta,
 				Spec: batch.JobSpec{
-					Selector: validGeneratedSelector,
-					Template: validPodTemplateSpecForGeneratedRestartPolicyNever,
+					Selector:             validGeneratedSelector,
+					Template:             validPodTemplateSpecForGeneratedRestartPolicyNever,
+					PodReplacementPolicy: &failedPodReplacement,
```

Comment:
leave this one nil (so we test that nil passes validation)

Reply:
Sorry, I find this a bit confusing..  

Do you mean leave it as:

```
PodReplacementPolicy: nil
```
or

Just drop the field entirely?

Reply:
they are equivalent and I prefer just dropping the field.

Reply:
Ah I see.  Pushed up the last of my changes.